packages/poll/poll.0.3.0/opam

```diff
@@ -0,0 +1,53 @@
+opam-version: "2.0"
+synopsis:
+  "Portable OCaml interface to macOS/Linux/Windows native IO event notification mechanisms"
+description:
+  "poll provides a portable OCaml interface to IO event notification mechanisms on macOS, Linux and Windows. It uses kqueue on macOS, epoll on Linux, and uses a vendored copy of wepoll on Windows."
+maintainer: ["Anurag Soni <anurag@sonianurag.com>"]
+authors: ["Anurag Soni"]
+license: "MIT"
+tags: ["epoll" "kqueue" "wepoll"]
+homepage: "https://github.com/anuragsoni/poll"
+doc: "https://anuragsoni.github.io/poll"
+bug-reports: "https://github.com/anuragsoni/poll/issues"
+depends: [
+  "dune" {>= "2.9"}
+  "base-unix"
+  "ppx_optcomp"
+  "kqueue" {>= "0.2.0"}
+  "dune-configurator"
+  "ppx_expect" {with-test}
+  "ppx_optcomp" {with-test}
```

Comment:
```suggestion
```
It is already required above

Reply:
Thanks! I'll fix the package definition in the `poll` repository.

Reply:
fixed at https://github.com/anuragsoni/poll/commit/34cec04efc263d2249ca7453bb88957c058a1834